client/layout/sidebar/expandable.jsx

```diff
@@ -56,12 +68,33 @@ export const ExpandableSidebarMenu = ( {
 	const classes = classNames( className, {
 		'is-toggle-open': !! expanded,
 		'is-togglable': true,
+		hovered: submenuHovered,
 	} );
 
+	const onEnter = () => {
+		if ( expanded || isTouch ) return;
```

Comment:
Let's use braces here and below, and break it out in multiple lines.
See https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#blocks-and-curly-braces

Reply:
Thanks @obenland for pointing this out! It was a really good time in recapping WP Coding Standards! 
Fixed 7f19464